src/components/ProjectProposedAllocationsTile.js

```diff
@@ -62,9 +72,9 @@ const ProjectProposedAllocationsTile = (props) => {
                             </Grid>
                             <Grid items xs={7}>
                                 <Typography color='secondary' variant='caption'>
-                                    {`${currencyInformation['symbol']}${a.rate.hourly_rate}/hr (
-                                    ${a.rate.type == 'monthly_rate' ? 'monthly rate' : 'max budget'}
-                                )`}
+                                    {`
+                                        ${currencyInformation['symbol']}${a.rate.hourly_rate}/hr (${a.rate.type == 'monthly_rate' ? 'monthly rate' : 'max budget'})
```

Comment:
This line is too long and we should enforce some max columns in the linter at some point in the future to prevent spaghetti code